bids/variables/kollekshuns.py

```diff
@@ -53,8 +54,14 @@ def __init__(self, variables):
                              "variables at more than one level of analysis. "
                              "Levels found in input variables: %s" %
                              var_levels)
-
-        self.level = list(var_levels)[0]
+        elif not var_levels:
+            raise ValueError(
+                "None of the provided variables matched any of the known levels, which are: %s"
+                % (', '.join(sorted(SOURCE_TO_LEVEL.values())))
+            )
+
+        try:
```

Comment:
Almost... You might also like to make use of 'git rebase -i' to make history cleaner

Reply:
actually, I did use git rebase -i HEAD~2 and used the f (fixup) option to put the cleanup under the original commit - or so I thought I did at least.